lib/docblockr-worker.js

```diff
@@ -100,7 +100,7 @@ module.exports =
         // extend line comments (// and #)
         else if((self.editor_settings.extend_double_slash == true) && (self.validate_request({preceding:true, preceding_regex:regex.extend_line, scope:'comment.line'}))) {
           // console.log('Snippet Extend line command');
-          var _regex = /^(\s*(?:#|\/\/[\/!]?)\s*).*$/;
+          var _regex = /^(\s*[^\sa-z0-9!]*\s*).*$/;
           var editor = atom.workspace.getActiveTextEditor();
```

Comment:
I don't understand why you included the `!` in the regex. Could you please remove it or explain why it is there?